apps/api/internal/mcp/connection_tools_test.go

```
@@ -0,0 +1,267 @@
+	}
+}
+
+// TestConnectionToolsIsA404ForAnUnknownConnection: absent, foreign and
+// RLS-refused all answer the same, or the difference is an oracle for which
+// ids exist.
+func TestConnectionToolsIsA404ForAnUnknownConnection(t *testing.T) {
+	store := &fakeStore{}
+	eng := newConnectionsRouter(t, store, orgPrincipal(uuid.New()))
+
+	w := httptest.NewRecorder()
+	eng.ServeHTTP(w, httptest.NewRequest(http.MethodGet, toolsPath(uuid.New()), nil))
```

golangci-lint (2.13.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)